packages/react/src/components/SkipLink/index.test.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import SkipLink from './';
+import axe from '../../axe';
+
+type SkipLinkProps = {
+  [key: string]: any;
+};
+
+const renderDefaultSkipLink = (props: SkipLinkProps = {}) => {
+  return render(
+    <SkipLink
+      target="#main-content"
+      skipText="Skip to"
+      targetText="Main page content"
+      {...props}
+    />
+  );
+};
+
+test('should focus main content when SkipLink is clicked', () => {
+  expect.assertions(2);
+
+  const target = window.document.createElement('div');
+  target.id = 'main-content';
+  window.document.body.appendChild(target);
```

Comment:
Just one small change here, we should probably but this fixture into before/after each hooks to ensure the global document cleans up after itself:

```ts
let target: HTMLDivElement

afterEach(() => {
  target = window.document.createElement('div');
  target.id = 'main-content';
  window.document.body.appendChild(target);
})

beforeEach(() => {
  window.document.body.removeChild(target);
})
```